NSubstitute.DbConnection/NSubstitute.DbConnection.Tests/QueryTests.cs

```diff
@@ -7,21 +7,30 @@
 
 namespace NSubstitute.DbConnection.Tests;
 
+using System.Data;
 using System.Data.Common;
 
 public class QueryTests
 {
-    [Test]
-    public void ShouldMockQuery()
+    private const string SimpleSelect = "select * from table";
```

Comment:
This is only used inside the private method so I would make it local within the method

Reply:
Was intended to replace other methods that use the same select, but I'm fine with this suggestion and this can always be done later.